changelog/17.0/17.0.0/summary.md

```diff
@@ -49,6 +50,13 @@ Here is a (condensed) example of stats output:
 }
 ```
 
+
+#### <a id="keyspace-name-validation"> Keyspace name validation in TopoServer
+
+Prior to v17, it was possible to create a keyspace with invalid characters, which would then be inaccessible to various cluster management operations.
+
+Keyspace names may no longer contain the forward slash ("/") character, and TopoServer's `GetKeyspace` and `CreateKeyspace` methods return an error if given such a name.
+
```

Comment:
Should we be backporting this change? This doesn't look like a bug-fix but a validation on the input received.